roles/postgres/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 download_folder: /opt
-postgres_repo_rpm_name: pgdg-centos96-9.6-3.noarch.rpm
-postgres_repo_download_url: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/{{postgres_repo_rpm_name}}
-postgres_repo_rpm_location: "{{download_folder}}/{{ postgres_repo_rpm_name}}"
+postgres_repo_rpm_name: pgdg-redhat-repo-42.0-11.noarch.rpm
+postgres_repo_download_url: https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/{{postgres_repo_rpm_name}}
```

Comment:
@buvaneswari-arun the old one which is "postgres_repo_download_url":
"https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64" works
i don't see the necessity of changing it

Reply:
Agreed. Reverting the changes.